### PR TITLE
Fix horizontal scroll for DataTable with fixed header

### PR DESCRIPTION
**Pull request by quadreex**

### CODE REVIEW

**Comment by quadreex** [comment]
@fspoettel I haven't touched DatePicker component. Do you know, why the tests are broken?

**Comment by fspoettel** [comment]
@quadreex Noticed that too, I'll investigate.

File: src/components/Table/Cell.js [hunk 1]
```diff
@@ -98,7 +98,7 @@ const Cell = (props) => {
   };
 
   const wrapChildren = (content) => {
-    if (!!sortable && isHeader) {
+    if (isHeader) {
```
Comment (fspoettel) [inline src/components/Table/Cell.js]: Shouldn't this return a span in case it is not a an action like `sortable`?

Comment (quadreex) [reply src/components/Table/Cell.js]: It breaks visual styles for non-sortable fields

**Pull request merged**